### ISSUE TITLE
Network create and destroy support

### ISSUE THREAD

**Issue by jdvr** [metacloud/molecule#1075]
# Issue Type

- Feature request

# Molecule and Ansible details

```
ansible --version
ansible 2.3.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.12 (default, Nov 20 2017, 18:23:56) [GCC 5.4.0 20160609]

molecule --version
molecule, version 2.5.0
```

- Molecule installation method: pip
- Ansible installation method: pip

# Desired Behaviour

Add option to declare docker network and add containers to those networks during platform declaration.

The idea is to have the same features that [molecule 1.25](https://github.com/idealista/redis-role/blob/d645f349795e0e03b4a31f0b37b9600fe1fcec8f/tests/cluster-mode/molecule.yml#L35).

example:
```yml
driver:
  name: docker
  networks:
    - name: foo
      bar: foobar
    - name: some
       bar: other
platforms:
  - name: vito.cont
    groups:
      - redis
    image: geerlingguy/docker-debian9-ansible:latest
    privileged: True
    capabilities:
      - SYS_ADMIN
    volumes:
      - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
    command: '/lib/systemd/systemd'
    networks:
       - foo
       - some
```

During create and destroy playbooks this data can be used along ansible `docker_network` to create and destroy networks.

### Example of manual implementation
* Create: https://github.com/idealista/redis-role/blob/master/molecule/cluster/create.yml#L35
* Destroy: https://github.com/idealista/redis-role/blob/d645f349795e0e03b4a31f0b37b9600fe1fcec8f/molecule/cluster/destroy.yml#L21

If it makes sense to you I could try to make a PR. This kind on scenarios are very common while testing services that works on cluster mode.

**Comment by retr0h** [comment]
> If it makes sense to you I could try to make a PR. This kind on scenarios are very common while testing services that works on cluster mode.

Sure, feel free to create a PR.

The intent of the create/destroy playbooks is for users to add whatever the like to these.  Sure, we can make molecule.yml abstract common patterns.

**Comment by jdvr** [comment]
I have problems running the test, is there any developer guide ?

**Comment by retr0h** [comment]
No

**Comment by dj-wasabi** [comment]
Attempt in pr #1097

**Comment by jdvr** [comment]
Thanks!

**Comment by retr0h** [comment]
Implemented by #1097.
